fractal/cmd/_task.py

```diff
@@ -143,12 +144,12 @@ async def patch_task(
     meta_file: Optional[str] = None,
 ) -> BaseInterface:
 
-    try:
-        task_id = int(task_id_or_name)
-        if version:
-            logging.warning("Task Version is ignored because Task ID provided")
-    except ValueError:
-        task_name = task_id_or_name
+    if task_id and version:
+        logging.warning(
+            "Too many arguments: cannot provide both `task_id` and `version`."
```

Comment:
Because it's now a warning, we should use a softer phrasing, like the original one (e.g. "`task_version` is ignored because `task_id` provided").